src/main/java/com.company/heap/RunningMedian.java

```diff
@@ -0,0 +1,101 @@
+package com.company.heap;
```

Comment:
Make sure to solve this in Leetcode and GFG also

Reply:
yes solved